### DISCUSSION TITLE
Custom chat info shared media screen

### DISCUSSION THREAD

**Discussion by JeromeCHA** [GetStream/stream-chat-android#3823] [Q&A] [answered]
Hi

I would like to know if it is possible to use and customize the Chat info shared media screen? I tried to look at the documentation, but could not find a way to do it.

Like, is it possible via the SDK to tap a button a show this screen with a different color on the toolbar, etc... ? 🙇‍♂️

| ChatInfo | ChatInfoSharedMediaFragment  |
|---|---|
| ![Screenshot_20220701_161330](https://user-images.githubusercontent.com/11457628/176843928-a7d8bd29-478f-450d-bc03-722cb08e024f.png) | ![Screenshot_20220701_160753](https://user-images.githubusercontent.com/11457628/176842991-e68f28b2-d5fa-4c0f-a57e-ad1fb9ccb243.png) |

**Comment by MarinTolic** [comment, marked as answer]
Hi Jerome,

The screens you are referring to are not a part of the SDK itself.

The UI Components sample app contains not only the parts of the SDK itself, but also custom built screens which are meant to showcase to our users how to achieve certain desired features. Such is the situation with `ChatInfoSharedMediaFragment` which uses `MediaAttachmentGridView` as a central piece that displays the media shared inside of a channel wrapped inside a pretty standard Android screen.

The easiest solution would be to recreate something similar to the screen you are referring to.

Kind regards,
Marin.

**Comment by JeromeCHA** [comment]
Hi @MarinTolic

I am sorry for bringing back this question, but is there any way we could get all media files from a Chat so we can recreate something similar?

Best regards,
Jerome
